e2e-cypress/files/package.json

```
@@ -14,19 +14,25 @@
     "junit-acceptance-report": "jrm build/test-results/acceptance-junit.xml 'build/test-results/acceptance-*.xml'",
     "delete-junit-results": "rimraf build/test-results",
     "e2e": "npm-run-all delete-junit-results cypress:run-installation cypress:run-integration cypress:run-acceptance",
+    "e2e:prod": "npm-run-all delete-junit-results cypress:run-installation",
     "combine:reports": "npm-run-all junit-installation-report junit-integration-report junit-acceptance-report",
-    "e2e:jenkins:record": "npm run delete-junit-results && npm run cypress:run-installation -- --record && npm run cypress:run-integration -- --record && npm run cypress:run-acceptance -- --record"
+    "e2e:jenkins:record": "npm run delete-junit-results && npm run cypress:run-installation -- --record && npm run cypress:run-integration -- --record && npm run cypress:run-acceptance -- --record",
+    "e2e:jenkins:report:prod": "npm run delete-junit-results && npm run cypress:run-installation -- --record",
```

I guess the intention here was to have a prod version of `e2e:jenkins:record`:
```suggestion
    "e2e:jenkins:record:prod": "npm run delete-junit-results && npm run cypress:run-installation -- --record",
```

Like for the `e2e:prod` the documentation (https://github.com/opendevstack/ods-quickstarters/blob/master/docs/modules/quickstarters/pages/e2e-cypress.adoc?plain=1#L124-L137) might require an update reflecting flavors for the different target environments.